components/button/ButtonSocialMedia.tsx

```diff
@@ -0,0 +1,25 @@
+import clsx from 'clsx'
+import { Image, ImageSourcePropType, Pressable, Text } from 'react-native'
+
+interface Props {
+  image: ImageSourcePropType
+  label?: string | undefined
```

Comment:
no es necesario indicar explicitamente que puede ser string o undefined, por defecto al usar label? se asigna el valor de tipo undefined
```suggestion
  label?: string //* string | undefined *//
```